### PR TITLE
add welcome file to codespaces

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

File: codespace-welcome.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Welcome to the TBIL Resource Library Codespace!
+
+(To see a prettier version of this file with clickable links, 
+tap `[Ctrl/Cmd]+[k]` then `[v]` on your keyboard.)
```
Comment (StevenClontz) [inline codespace-welcome.md:4]:
```suggestion
tap `[Ctrl/Cmd]+[Shift]+[v]` on your keyboard.)

```

File: codespace-welcome.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+To begin editing a book, open the appropriate file:
+
+- Calculus: [`codechat_config_calc.yaml`](codechat_config_calc.yaml)
+- Preclculus: [`codechat_config_precal.yaml`](codechat_config_precal.yaml)
+- Linear Algebra: [`codechat_config_la.yaml`](codechat_config_la.yaml)
+
+Copy its entire contents, then open and overwrite *ALL* the contents of
+the [`codechat_config.yaml`](codechat_config.yaml) file.
+
```
Comment (siwelwerd) [inline codespace-welcome.md:14]:
```suggestion

Alternatively, open a terminal and type `cp codechat_config_BOOK.yaml codechat_config.yaml`, replacing BOOK with either 'calc', 'precal', or 'la'.

```

**Review by siwelwerd — APPROVED** [review]

**Comment by siwelwerd** [comment]
If we can make Codespaces open up a markdown file, can we make Codespaces open up a little ipywidget dashboard with a dropdown, that runs the `cp` upon selection?

**Comment by StevenClontz** [comment]
> If we can make Codespaces open up a markdown file, can we make Codespaces open up a little ipywidget dashboard with a dropdown, that runs the `cp` upon selection?

> ipywidget dashboard

ew 🤣 What idiot would use ipywidgets as a UI? 🙃 

To the point though, creating a UI to set up a Codespace is intriuging, but out of scope of what I think we can accomplish this week.

**Comment by StevenClontz** [comment]
I think shipping a custom VS Code extension, that's just part of the repo and installed from the repo itself, might be the right way to add custom UI features to our codespace.

**Pull request merged**